extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -48,9 +48,13 @@ amp-embed iframe {
   padding: 2px 4px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type] > iframe {
```

Comment:
Good point regarding other networks. I will narrow this down to AdSense and DoubleClick. For the moment, however, I don't have a better selector than this data attribute.